src/templates/apiPage.tsx

```diff
@@ -77,7 +72,7 @@ export default props => {
     apiExample.push(` -H 'Content-Type: ${contentType}'`);
   }
 
-  if (bodyParameters) {
+  if (bodyParameters.length > 0) {
```

Comment:
Nit: we can remove the `>0` since `0` is falsy in javascript

Reply:
Good catch!